src/core/script/disclosure/disclosures-group.js

```diff
@@ -66,7 +66,9 @@ class DisclosuresGroup extends Instance {
     return this._members;
   }
 
-  get length () { return this.members.length; }
+  get length () {
+    return this.members ? this.members.length : null;
```

Comment:
```suggestion
    return this.members ? this.members.length : 0;
```

Checker Accordions, Modal, Tabs, Navigation fonctionne bien.